src/blurredLocation.js

```diff
@@ -8,13 +8,21 @@ BlurredLocation = function BlurredLocation(options) {
   options.pixels = options.pixels || 400;
 
   options.gridSystem = options.gridSystem || require('./core/gridSystem.js');
+  options.Interface = options.Interface || require('./ui/Interface.js');
 
   gridSystemOptions = options.gridSystemOptions || {};
   gridSystemOptions.map = options.map;
   gridSystemOptions.gridWidthInPixels = gridWidthInPixels;
   gridSystemOptions.getMinimumGridWidth = getMinimumGridWidth;
   gridSystem = options.gridSystem(gridSystemOptions);
 
+  InterfaceOptions = options.InterfaceOptions || {};
+  InterfaceOptions.Id = {latId: 'lat', lngId: 'lng'}
```

Comment:
Done, but we have to keep the panMap method in the options as it is imported from this file itself